pom.xml

```diff
@@ -38,7 +38,7 @@
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
         <ddf.version>2.11.0-SNAPSHOT</ddf.version>
-        <ddf.support.version>2.3.6</ddf.support.version>
+        <ddf.support.version>2.3.7-SNAPSHOT</ddf.support.version>
```

Comment:
It will require a release of 2.3.7 prior to releasing the next version off master but to test it out first, it needs to be against the snapshot.

Presumably the next release of Alliance off of master is far enough out that it is safe if it takes few days or a week to verify and release support. If the next Alliance release off of the master branch is slated to go sooner, this should not be merged until after then.